discovery.md

```diff
@@ -110,6 +117,76 @@ be considered an event subscriber and a target user of this API.
 This API is specified as a REST API with well defined entities and relationships
 between those entities.
 
+To help explain the data model, the following non-normative pseudo yaml shows
+its basic conceptual structure:
+
+( `*` means zero or more, `+` means one or more, `?` means optional)
+
+```
+- services: *
+  - service: [unique identifier for producer of the specified event types]
```

Comment:
If I recall correctly, one of the proposals before we agreed on source was something like "service"+"resource". 😄